_includes/configuration/resize.md

```diff
@@ -2,6 +2,11 @@
 
 This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `'both'`. `False` disables the resize, `true` enables vertical resizing only, `'both'` makes it possible to resize in both directions horizontal and vertical.
 
+When resizing is enabled, the editor can be resized by either:
+
+* Clicking and dragging the resize handle in the bottom right-hand corner.
+* Focusing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smaller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```

Comment:
I guess I was thinking of something more specific, but also that would be complicated with the different resizing modes, so I guess this works.